<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingWindow</class>
 <widget class="QMainWindow" name="settingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>160</width>
      <height>360</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="button_person">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>160</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#button_person::checked{
    color:#f00;
    background-color:#F1E7E6;
    border:none;
}

QPushButton#button_person{
    color:#000;
    border:none;
    }
QPushButton#button_person::hover{
    color:#f00;
	
    border:none;
    }</string>
     </property>
     <property name="text">
      <string>个人信息</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_vip">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>160</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#button_vip::checked{
    color:#f00;
    background-color:#F1E7E6;
    border:none;
}

QPushButton#button_vip{
    color:#000;
    border:none;
    }
QPushButton#button_vip::hover{
    color:#f00;
	
    border:none;
    }</string>
     </property>
     <property name="text">
      <string>会员中心</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_setting">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>160</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#button_setting::checked{
    color:#f00;
    background-color:#F1E7E6;
    border:none;
}

QPushButton#button_setting{
    color:#000;
    border:none;
    }
QPushButton#button_setting::hover{
    color:#f00;
	
    border:none;
    }</string>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_manyi">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>160</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#button_manyi::checked{
    color:#f00;
    background-color:#F1E7E6;
    border:none;
}

QPushButton#button_manyi{
    color:#000;
    border:none;
    }
QPushButton#button_manyi::hover{
    color:#f00;
	
    border:none;
    }</string>
     </property>
     <property name="text">
      <string>满意度调查</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_feedback">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>160</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#button_feedback::checked{
    color:#f00;
    background-color:#F1E7E6;
    border:none;
}

QPushButton#button_feedback{
    color:#000;
    border:none;
    }
QPushButton#button_feedback::hover{
    color:#f00;
	
    border:none;
    }</string>
     </property>
     <property name="text">
      <string>意见反馈</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_about">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>160</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#button_about::checked{
    color:#f00;
    background-color:#F1E7E6;
    border:none;
}

QPushButton#button_about{
    color:#000;
    border:none;
    }
QPushButton#button_about::hover{
    color:#f00;
	
    border:none;
    }</string>
     </property>
     <property name="text">
      <string>关于</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>631</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_person">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>QQ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>修改信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>更改</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>150</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>生    日:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>介    绍:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>登录方式:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_6">
      <property name="geometry">
       <rect>
        <x>295</x>
        <y>240</y>
        <width>150</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>240</y>
        <width>150</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>昵    称:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>481</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>这家伙很懒,什么也没留下</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>头    像:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>编辑个人信息</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>480</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>crush</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>头像地址</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_vip"/>
    <widget class="QWidget" name="page_setting">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>631</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane{
    border:none;
    
}

QTabBar::tab{
	background:transparent;
	color:#000;
 
}

QTabBar::tab:selected{
	color:#f00;
	

}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_setting_normal">
       <attribute name="title">
        <string>常规</string>
       </attribute>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>字体选择：</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="tab_setting_normal_groupBox_font">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QRadioButton" name="radioButton_big">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>115</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>  较大</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_normal">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>115</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string> 标准</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>启动：</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="tab_normal_startup_bootup">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>351</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>开机时自动启动</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="tab_normal_startup_min">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>361</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>启动后最小化到系统托盘</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="tab_normal_mainwindow_first">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>381</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>启用主窗口在前</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="tab_normal_mainwindow_min">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>371</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>窗口关闭时最小化到系统托盘</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="tab_normal_rightClick_add">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>390</y>
          <width>361</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>把使用有道词典翻译添加到鼠标右键菜单</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>主窗口：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>文档翻译：</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>离线增强</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>内容</string>
       </attribute>
       <widget class="QListWidget" name="listWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>781</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>查词时自动发音：</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="tab_content_audio">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>571</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>启动自动发音（按下回车和点击查询按钮时有效）</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>查询历史记录：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>选择查询历史记录的条数（0-1000）：</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="tab_content_history_count">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>200</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="tab_content_clear_history">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>清空历史</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>取词划词</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>单词本</string>
       </attribute>
       <widget class="QListWidget" name="listWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>241</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>每日进入复习流程的新单词上限：</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="tab_note_checkBox_noteadd">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>491</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>查过的词自动加入单词本（按下回车和点击&quot;查询&quot;按钮时有效）</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="tab_note_checkBox_review">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>491</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>单词复习时自动发音</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="tab_note_checkBox_reviewadd">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>491</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>添加单词时默认加入复习计划</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>370</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>个</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="tab_note_checkBox_browse">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>491</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>单次浏览时发音</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>400</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>添加单词时默认添加到：</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="tab_note_checkBox_showexplain">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>491</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>单词播放时直接显示释义</string>
        </property>
       </widget>
       <widget class="QComboBox" name="tab_note_comboBox">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>400</y>
          <width>301</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="tab_note_checkBox_remind">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>491</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>在系统任务栏（系统托盘）提醒复习</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="tab_note_lineEdit">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>370</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>网络</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_manyi"/>
    <widget class="QWidget" name="page_feedback"/>
    <widget class="QWidget" name="page_about"/>
   </widget>
   <widget class="QPushButton" name="button_close">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="toolTip">
     <string>关闭</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#button_close{
	
	border-image: url(:/images/close.png);
	border:none;
}

#button_close::hover{
	
	border-image: url(:/images/close-hover.png);
	border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
